it/uk/gov/hmrc/agentsubscription/controllers/SubscriptionControllerISpec.scala

```diff
@@ -293,7 +293,166 @@ class SubscriptionControllerISpec extends BaseISpec with DesStubs with AuthStub
     }
   }
 
+  "updating a partial subscription" should {
+    "return a response containing the ARN" when {
+      "a valid utr is given as input" in {
+        requestIsAuthenticated().andIsAnAgent().andHasNoEnrolments()
+        agentRecordExists(utr, true, arn)
+        allocatedPrincipalEnrolmentNotExists(arn)
+        deleteKnownFactsSucceeds(arn)
```

Comment:
again, can you check whether we can do this? I do not think we can override with new known facts